src/shell/autoexec.cpp

```diff
@@ -406,7 +410,7 @@ AutoExecModule::AutoExecModule(Section* configuration)
 			LOG_WARNING("AUTOEXEC: Secure mode skipped as it does not allow booting");
 		} else {
 			AddLine(Placement::SecureModeCommand,
-			        "@Z:\\CONFIG.COM -securemode");
+			        CmdConfig + std::string("-securemode"));
```

Comment:
std::string can add literals, so maybe a simplification to avoid the `std::string(...)`

Reply:
Ok, changed.